src/framework/uicomponents/qml/MuseScore/UiComponents/StyledDialogView.qml

```diff
@@ -63,7 +65,9 @@ DialogView {
     }
 
     onOpened: {
-        Qt.callLater(navSec.requestActive)
+        if (root.isDoActiveFirstControlOnOpen) {
+            Qt.callLater(navSec.requestActive)
```

Comment:
The activate of section need for Escape from dialog, because of it we should always activate section.
This if is redundant.

Reply:
Removing the if breaks the feature of the issue

Reply:
why?

Reply:
The search bar is not automatically in focus when the new score dialog is opened. I don't know the exact reason, but when a new score dialog is opened, the isDoActiveFirstControlOnOpen variable is false, so the callLater function is not called and the focus is correctly on the search bar